Fixing spacing between vision and console components to match the design

AdamRathbun opened the pull request:
Hi, new to Tailwind, but I looked at the Figma design and changed the styling to match. Let me know if you'd like it differently!

Akalanka47000 reviewed (changes requested):
Hi @AdamRathbun , the changes visually looks good. There are 2 things which can be changed though, it'll be nice if we can get them in

- The project uses [pnpm](https://pnpm.io/installation) as a package manager. Better to remove the package-lock.json file
- Using negative margins can create overlapping at certain times, would be nice if we reduce the margin bottom from the place it's initially added rather than adding a negative margin to cancel it out.

AdamRathbun commented:
Hi @Akalanka47000, I deleted the package-lock json file and changed the spacing fix by taking away the margin top from the vision component and then changing the console component spacing to match. I tried to reduce the spacing more but could not find where the extra spacing is coming from. Would you like the spacing to be reduced even more?

Akalanka47000 commented:
> Hi @Akalanka47000, I deleted the package-lock json file and changed the spacing fix by taking away the margin top from the vision component and then changing the console component spacing to match. I tried to reduce the spacing more but could not find where the extra spacing is coming from. Would you like the spacing to be reduced even more?

This looks good, I'll tweak it a bit if needed. Just for your information you can easily find out where an element's spacing comes from from the inspect element menu. We just need to go through the DOM tree - 

![image](https://github.com/sliit-foss/timekeeper-web/assets/73662613/d312b5b7-3bc7-4daa-938c-fb1d0549e4d4)

![image](https://github.com/sliit-foss/timekeeper-web/assets/73662613/ea9eecbf-2d71-475a-a159-3c24f72f0d9d)

Thanks once again for the PR

AdamRathbun commented:
Thanks @Akalanka47000, I enjoyed working on this
